test/test_tree_2d_part2.jl

```diff
@@ -21,6 +21,9 @@ isdir(outdir) && rm(outdir, recursive=true)
 
   # Compressible Euler Multicomponent
   include("test_tree_2d_eulermulti.jl")
+
+  # Compressible Euler coupled with acoustic perturbation equations
+  include("test_tree_2d_euleracoustics.jl")
```

Comment:
How long does this test take and does it keep the approximately uniform time distribution of the tests?

Reply:
It takes [142 seconds](https://github.com/trixi-framework/Trixi.jl/pull/712/checks?check_run_id=3574900220#step:7:11031) However, most if it seems to be compilation: If I run the same setup (`include("test_tree_2d_euleracoustics.jl")`) on Roci, I get 8 seconds for the subsequent runs.

In terms of uniformity, it seems to be ok:
![image](https://user-images.githubusercontent.com/3637659/132981856-d7eddd35-2d23-48c3-a881-dfc275d80b4b.png)